CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### 🐞 Fixed
 - Fix encoding of channels with custom type [#872](https://github.com/GetStream/stream-chat-swift/pull/872)
+- Fix currentUser returning nil if `synchronize` was not called [#875](https://github.com/GetStream/stream-chat-swift/pull/875)
```

Comment:
Changelog should be written from the user's perspective. So this should be something like `Fix message actions not visible on long press`

Reply:
Yes the reason I did this is because although we identified the issue in the actions. It is actually a broader problem and is related with LLC. If someone was using LLC now, the messages hidden would not make sense if they were looking to fix an issue where they could not get the current user. 🤔

Reply:
I see, in that case, it could be something like: 
`Fix `CurrentUserController.currentUser` incorrectly returning `nil` before `synchronize` is called`?

Reply:
Yes that is a bit more clear, I'll change it 👍

Reply:
✅